gdal/ci/travis/osx/install.sh

```diff
@@ -63,3 +63,6 @@ cd ../../gdal
 ln -s /tmp/install/lib/libinternalproj.15.dylib /tmp/install/lib/libproj.15.dylib
 
 ccache -s
+
+# Post-install testing
+./autotest/postinstall/test_pkg-config.sh /tmp/install
```

Comment:
due to line 63 (which could be changed) the current directory is the gdal subdirectory, not the root of the repository